configs/native-lookup.yaml

```diff
@@ -4,6 +4,7 @@ plugin_groups:
     - path: pose_lookup
     - path: gldemo/
     - path: debugview/
+    - path: offload_data/
```

Comment:
This plugin can always be loaded, while the `enable_offload` flag handles the real activation/deactivation.
This plugin introduces barely zero overhead if we set the flag to 0. This simplifies the procedure from `activating the flag + load the plugin` to `activate the flag` only.